app/data/bibtex_to_ingest/August19 17 Batch/aydin.txt

```diff
@@ -53,7 +53,7 @@ journal = {Journal of Structural Geology},
 title = {Growth processes, dimensional parameters and scaling relationships of two conjugate sets of compactive shear bands in porous carbonate grainstones, Favignana Island, Italy},
 year = {2012},
 doi = {10.1016/j.bbr.2011.03.031},
-url = {http://dx.doi.org/10.1016/j.bbr.2011.03.031},
+url = {https://dx.doi.org/10.1016/j.bbr.2011.03.031},
 }

```

Comment:
Tried both of these in a browser, which does auto-redirection, and they both worked.  So these changes are probably OK.